services/director-v2/src/simcore_service_director_v2/core/dynamic_services_settings/sidecar.py

```diff
@@ -50,6 +49,46 @@ def enforce_r_clone_requirement(cls, v: int, values) -> PositiveInt:
         return v
 
 
+class PlacementConstraintStr(ConstrainedStr):
```

Comment:
Please consider moving this class to packages/models-library/src/models_library/docker.py

We have there already DockerLabelKey and DockerGenericTag

And also consider renaming to something like ```DockerPlacementConstraint``` also I am not sure how different this is from ```DockerLabelKey``` in terms of regex. please check, maybe you can just re-use it.

Reply:
could not reuse the `DockerLabelKey`, will keep the current one.